lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -855,11 +888,13 @@ sub fix_tests{
     for my $count (1 .. $request->{count}){
         my $id = $request->{"id_$count"};
         my @values;
+        push @values, $id
+            if $check->{insert} and $id_displayed;
```

Comment:
the value of `$check->{insert}` determines the value of `$id_displayed`. No need to check it again.